server/src/app/index.js

```diff
@@ -0,0 +1,34 @@
+import {
+  express,
+  index,
+  users,
+  mongoose,
+  dotenv,
+  bodyParser
+} from "./constants/packageConfig";
+
+dotenv.config({ path: "../.env" });
+
+const port = process.env.PORT || 5000;
```

Comment:
Move env variables to the constant/config file.

Reply:
Done